test/integration/monitoring/update-deliveries.test.js

```diff
@@ -178,12 +178,12 @@ describe('update deliveries', () => {
   test('should send email to requested email address if status technical failure', async () => {
     mockGetNotificationById = jest.fn().mockResolvedValue({ data: { status: TECHNICAL_FAILURE } })
     await updateDeliveries()
-    expect(mockSendEmail.mock.calls[0][1]).toBe('farmer1@farm.com')
+    expect(mockSendEmail.mock.calls[0][1]).toBe(mockStatement1.email)
   })
 
-  test('should send email with file link if status technical failure', async () => {
-    mockGetNotificationById = jest.fn().mockResolvedValue({ data: { status: TECHNICAL_FAILURE } })
-    await updateDeliveries()
-    expect(mockSendEmail.mock.calls[0][2].personalisation.link_to_file).toBe(MOCK_PREPARED_FILE)
-  })
+  // test('should send email with file link if status technical failure', async () => {
+  //   mockGetNotificationById = jest.fn().mockResolvedValue({ data: { status: TECHNICAL_FAILURE } })
+  //   await updateDeliveries()
+  //   expect(mockSendEmail.mock.calls[0][2].personalisation.link_to_file).toBe(MOCK_PREPARED_FILE)
+  // })
```

Comment:
Remove if not needed.

Reply:
Good spot -- removed to reduce the output from logs when recording the Notify response object.

Added back in and all passing